components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/utils/SCIMCommonConstants.java

```diff
@@ -104,6 +104,8 @@
     // Date of Birth related constants.
     public static final String DATE_OF_BIRTH_LOCAL_CLAIM = "http://wso2.org/claims/dob";
     public static final String PROP_REG_EX = "RegEx";
+    public static final String PROP_REG_EX_VALIDATION_ERROR = "RegExValidationError";
+    public static final String DOB_REG_EX_VALIDATION_DEFAULT_ERROR = "Date of Birth is not in the correct format";
```

Comment:
put a sample format YYYY-MM-DD  Ex. 1998-12-24

Reply:
we need to add this to the default claim config.

Reply:
added https://github.com/wso2/carbon-identity-framework/pull/3383